modules/candidate_parameters/test/TestPlan.md

```diff
@@ -1,8 +1,9 @@
 # Candidate Parameters Test Plan
 
 1. Check Permissions. Try removing each of these permissions/conditions one at a time. You should not be able to access the page
-	* data_entry
-	* unsend_to_dcc
+    * Edit Candidate Parameters
```

Comment:
@johnsaigle We have an Edit Candidate Parameters and View Candidate Parameters permission now.  
These are the only permissions that should matter for this module.  If the code checks these (and not data_entry and unsend_to_dcc), can you update the test plan to reflect this?  If the code doesn't use them properly/fully, could you add a ticket in Redmine?